python/cugraph/cugraph/structure/number_map.py

```diff
@@ -648,6 +666,31 @@ def unrenumber(self, df, column_name, preserve_order=False, get_column_names=Fal
                 mapping[nm] = nm + "_" + column_name
             col_names = list(mapping.values())
 
+        if isinstance(self.input_src_col_names, list):
+            input_src_col_names = self.input_src_col_names.copy()
+            input_dst_col_names = self.input_dst_col_names.copy()
+        else:
+            # Assuming the src and dst columns are of the same length
+            # if they are lists.
+            input_src_col_names = [input_src_col_names]
+            input_dst_col_names = [input_dst_col_names]
+        if not isinstance(col_names, list):
+            col_names = [col_names]
+
+        # FIXME: instead of hardcoded value, it should be 'simpleGraphImpl.srcCol'
+        # but there is no way to retrieve it with the current API
+        if column_name in [self.renumbered_src_col_name, "src"]:
```

Comment:
I thought we had "src" and "dst" defined somewhere so that we always used a reference rather than a string that we could type wrong

Reply:
Yes we had it defined in `simpleGraph` but I didn't see that it was also store in NumberMap until @rlratzel pointed that out. It should be fixed in the next commit

Reply:
Actually no, those are not reference in NumberMap. Now I remember why I added the FIXME stating that we can't retrieve those column names. `self.src_col_names` and `self.dst_col_names` are the column names passed by the user as input and are not the internal referenced ones in `simpleGraphImpl.srcCol` and `simpleGraphImpl.dstCol`. With our current API, those ones are not retrievable in NumberMap.